src/modules/taskFunctions.js

```diff
@@ -41,10 +43,12 @@ class Tasks {
     });
   };
 
-  delete = (index) => {
+  deleteListItem = (index) => {
     this.tasks = this.tasks.filter((task) => task.index !== Number(index) + 1);
   };
 }
 
-const taskRemaining = new Tasks();
-export default taskRemaining;
\ No newline at end of file
+export const taskRemaining = new Tasks();
```

Comment:
- [ ] Nice work making the change requested by the previous reviewers :clap: However the app is not working because of a bug on line 50. The `taskRemaining` was imported as default in the `add.js` but here is not exported as default. Kindly fix it.
![Screenshot from 2023-06-15 21-24-37](https://github.com/kkmanuu/To-Do-Lists/assets/91142432/3086eb25-d90b-49cc-9920-f37cb8812382)

Reply:
@LionRouge1  Done